torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12134,7 +12134,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            sample_inputs_func=sample_inputs_layer_norm,
            supports_expanded_weight=True,),
     OpInfo('nn.functional.local_response_norm',
-           dtypes=floating_types_and(torch.int64, torch.bfloat16),
+           dtypes=floating_types_and(torch.int64, torch.float16, torch.bfloat16),
```

Comment:
Is half supported for avgpool3d? LRN would also invoke avgpool3d?

Reply:
LRN reused AvgPool2d kernel. AvgPool1d and AvgPool2d used the same set of kernel. AvgPool2d and AvgPool3d have different kernels.